test/chpldoc/compflags/creole/module.doc.catfiles

```diff
@@ -0,0 +1,6 @@
+docs/M.doc.txt
+docs/N.doc.txt
+docs/O.doc.txt
+docs/P.doc.txt
+docs/Q.doc.txt
+docs/R.doc.txt
```

Comment:
I don't think the ".doc" suffixes are necessary.  The similar test which lives at module/module.chpl didn't need the .doc for all these sibling modules.  Testing wouldn't have found this because the creole ones don't run anymore (and I need to convert them to .rst checking ones anyways)